static/src/javascripts/projects/commercial/modules/creatives/page-skin.js

```diff
@@ -8,11 +8,10 @@ import { commercialFeatures } from 'common/modules/commercial/commercial-feature
 const pageSkin = (): void => {
     const bodyEl = document.body;
     const hasPageSkin: boolean = config.get('page.hasPageSkin');
-
     const isInAUEdition = config.get('page.edition', '').toLowerCase() === 'au';
     const adLabelHeight = 24;
-
     let topPosition: number = 0;
+    let hasTruskin: boolean = false;
```

Comment:
The behaviour of this property has changed quite a bit, is there a better name we could give it now?

Previously it was determined by what was in the DOM, so it made sense to say "has truskin". It looks to me like this is now determined by whether or not a `truskinRendered` postmessage has been received. A name that helps the reader see this will be changed by a postmessage may make this more understandable. Some suggestions off the top of my head:

* `truskinRendered`
* `truskinHasRendered`
* `truskingRenderEventRecieved`

This is a very minor point, I don't feel strongly about it! See what you think :-)

Reply:
I think this is a great point especially for preventing code rot when people come to  work on this in the future.